src/github.com/getlantern/flashlight/ui/ui.go

```diff
@@ -0,0 +1,72 @@
+package ui
+
+import (
+	"fmt"
+	"net"
+	"net/http"
+	"path"
+	"path/filepath"
+
+	"github.com/getlantern/golog"
+	"github.com/getlantern/tarfs"
+	"github.com/skratchdot/open-golang/open"
+)
+
+const (
+	localResourcesPath = "../ui/app"
```

Comment:
I think this was intended for @atavism

On Fri, Feb 20, 2015 at 10:22 AM, oxtoacart <notifications@github.com>
wrote:

> In src/github.com/getlantern/flashlight/ui/ui.go
> <https://github.com/getlantern/flashlight-build/pull/62#discussion_r25090293>
> :
>
> > +package ui
> > +
> > +import (
> > +	"fmt"
> > +	"net"
> > +	"net/http"
> > +	"path"
> > +	"path/filepath"
> > +
> > +	"github.com/getlantern/golog"
> > +	"github.com/getlantern/tarfs"
> > +	"github.com/skratchdot/open-golang/open"
> > +)
> > +
> > +const (
> > +	localResourcesPath = "../ui/app"
>
> @myleshorton <https://github.com/myleshorton> Just changed this to make
> it relative to the GOPATH so it should work no matter where you run
> flashlight. Can you please try again?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/getlantern/flashlight-build/pull/62/files#r25090293>.
>



-- 
--
President
Brave New Software Project, Inc.
https://www.getlantern.org
pgp A998 2B6E EF1C 373E 723F A813 045D A255 901A FD89